src/Target.h

```diff
@@ -140,6 +140,11 @@ struct Target {
         return has_feature(CUDA) || has_feature(OpenCL);
     }
 
+    /** Some device apis use coordinate-based stores/loads. */
+    bool has_coordinates_feature() const {
+        return has_feature(OpenGL);
```

Comment:
@dsharletg , well inside_kernel_loop condition is finer-grained since it looks for the loops that are scheduled for appropriate target device(OpenGL), while this has_coordinates_feature guards whole pass. Basically inject_coordinates_intrinsics does zero_gpu_loop_mins in addition to stuff in inside_kernel_loop.
I was using existing OpenGL flow as inspiration, truth be told...